src/ol_infrastructure/infrastructure/operations/consul.py

```diff
@@ -52,7 +51,6 @@
 consul_instance_profile = iam.InstanceProfile(
     f"consul-instance-profile-{environment_name}",
     role=consul_instance_role.name,
-    name=f"consul-instance-profile-{environment_name}",
     path="/ol-operations/consul/profile/",
```

Comment:
Why the changes here?

Reply:
Because it was giving me an error when trying to replace the resource due to name conflicts.